### PR TITLE
LP-2303 Handling multiple objects returned by AnonymousUserId

### PR DESCRIPTION
**Pull request by zahid-arbisoft**
### Description

[LP-2303](https://philanthropyu.atlassian.net/browse/LP-2303) Auto score ORA command

### Notes

If `AnonymousUserId` model have more than one anonymous user ids against a combination of user and course then do not auto score, to avoid a crash.

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Review by muak50 — APPROVED** [review]
LGTM 👍

File: openedx/features/assessment/helpers.py [hunk 1]
```diff
@@ -66,7 +67,15 @@ def _get_submissions_to_autoscore_by_enrollment(enrollment, submission_uuids, de
         list: All submissions in enrollment matching criteria
     """
     course_id = enrollment.course_id
-    anonymous_user_id = enrollment.user.anonymoususerid_set.get(course_id=course_id).anonymous_user_id
+    anonymous_user = get_anonymous_user(enrollment.user, course_id)
+
+    if not anonymous_user:
+        # If AnonymousUserId model have more than one anonymous user ids against a combination of user and course then
```
Comment (ahmed-faraz46) [inline openedx/features/assessment/helpers.py:73]: Let's please make it a `TODO` comment.

**Review by ahmed-faraz46 — CHANGES_REQUESTED** [review]

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM 👍

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Pull request merged**